### PR TITLE
commit pr

### PR DESCRIPTION
**Pull request by JulianBharadwaja**

### CODE REVIEW

File: 1_assignment.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+// METHOD WAY
+
+const calculatePrices = (itemArr) => {
+    let totalPriceArray = itemArr.map(item => item.price - (item.price * (item.discountPercent/100)));
+    let total = 0
+    let totalPrice = totalPriceArray.forEach(price => {
+        total += price;
+    });
+    console.log(total + " USD")
```
Comment (msquareprogramming) [inline 1_assignment.js]: The function is correct, but it should return the final price in string form such as "132 USD".

File: 1_assignment.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+// LOOPING WAY
+const calculateNormalPrices = (itemArr) => {
+    
+    let total = 0;
+    for (let i = 0; i < itemArr.length; i++) {
+        total = total + itemArr[i].price - (itemArr[i].price * itemArr[i].discountPercent/100);
+    }
+    console.log(total + " USD");
```
Comment (msquareprogramming) [inline 1_assignment.js]: [See this comment](https://github.com/JulianBharadwaja/JavaScript-HighGround-Assigment/pull/1/files#r671837090)

File: 1_assignment.js (hunk 1 above)
Comment (JulianBharadwaja) [reply 1_assignment.js]: alright Oh i get it :D

**Review by msquareprogramming — APPROVED** [review]
Looks good to me! :)

**Pull request merged**